vnext/local-cli/generate-windows.js

```diff
@@ -26,7 +26,7 @@ function generateWindows (projectDir, name, ns, options) {
     path.join(__dirname, 'generator-windows', 'templates'),
     projectDir,
     name,
-    { ns, overwrite: options.overwrite, language: options.language, experimentalNugetDependency: options.experimentalNugetDependency }
+    { ns, overwrite: options.overwrite, language: options.language, experimentalNugetDependency: options.experimentalNuGetDependency }
```

Comment:
FYI @dannyvv we should be careful in the future about changing anything at the generator boundary, since new versions of react-native-windows-init need to work with old versions of react-native-windows. I have a big warning at the TS interface for Generator Options now, but generally we probably rename/remove old things.

Reply:
I performed this rename after it was pointed out the name was NuGet instead of Nuget. 
Note there are many more locations that I updated the flag: `experimentalNugetDependency` to `experimentalNuGetDependency` than this line... Shall I restore all?

---
In reply to: [448407233](https://github.com/microsoft/react-native-windows/pull/5401#discussion_r448407233) [](ancestors = 448407233)

Reply:
-or- perhaps we can add both, and add a 'hidden' wrong casing one. Let me know and I can fix that.

---
In reply to: [448459254](https://github.com/microsoft/react-native-windows/pull/5401#discussion_r448459254) [](ancestors = 448459254,448407233)

Reply:
I don't think compat matters too much for this one in particular because we didn't publicly expose it.